### DISCUSSION TITLE
请问JuiceFS与云上的文件存储网关在使用场景上有什么区别？

### DISCUSSION THREAD

**Discussion by zhaojinxin409** [juicedata/juicefs#584] [General]
比如阿里云的CSG：https://help.aliyun.com/document_detail/53972.html?spm=a2c4g.11174283.2.5.649b4c071u6crS&aly_as=UGCnCBR5

刚接触JuiceFS，对产品还不太了解:）

**Comment by xiaogaozi** [comment]
CSG 是一个兼容 NFS/SMB 协议的网关，文件网关基于 OSS，从下表的数据上看只能用于小数据量的场景，相比之下 JuiceFS 能够应用的数据规模更大（PB 级容量，上千客户端）。

![image](https://user-images.githubusercontent.com/149645/124101750-3ee1d500-da92-11eb-91ec-9abf4a0b15f1.png)

**Reply by zhaojinxin409** [reply]
了解了， thanks
